content/acra/configuring-maintaining/metrics/_index.md

```diff
@@ -105,6 +105,37 @@ Additional metrics registration info could be found in [`prometheus.go`](https:/
 
     **Labels**: `edition`, `version`
 
+11. **Metric**: *acra_decryptions_total* (Available since 0.94.0)
+
+    **Description**: Number of decryptions AcraStruct/AcraBlock
+
+    **Type**: [`Counter`](https://prometheus.io/docs/concepts/metric_types/#counter)
+
+    **Labels**: `status`, `type`
+
+12. **Metric**: *acra_encryptions_total* (Available since 0.94.0)
+
+    **Description**: Number of encryptions AcraStruct/AcraBlock
+
+    **Type**: [`Counter`](https://prometheus.io/docs/concepts/metric_types/#counter)
+
+    **Labels**: `status`, `type`
+
+13. **Metric**: *acra_tokenizations_total* (Available since 0.94.0)
+
+    **Description**: Number of tokenizations for token_type
+
+    **Type**: [`Counter`](https://prometheus.io/docs/concepts/metric_types/#counter)
+
+    **Labels**: `status`, `token_type`
+
+14. **Metric**: *acra_detokenizations_total* (Available since 0.94.0)
+
+    **Description**: Number of detokenizations for token_type
+
+    **Type**: [`Counter`](https://prometheus.io/docs/concepts/metric_types/#counter)
+
+    **Labels**: `status`, `token_type`
 
 ### Metrics example
```

Comment:
can you make all types of queris to track all types of metrics on the acra-server side and update example with all of them? to have one place with example of all kinds of metrics. additionally it will show `token_type` and `type` values in context of each metric